### ISSUE TITLE
Let IntegerVersionField starts with '1'

### ISSUE THREAD

**Issue by shintophilip** [saxix/django-concurrency#150]
Suppose we have a model,

    class User(models.Model):
        version = AutoIncVersionField()

        id = models.UUIDField(primary_key=True, default=uuid.uuid4)
        name = models.CharField(max_length=255)

If we create the object using create, we are getting the expected version value(which is 1).

    User.objects.create(name='name')

But instead of that if we create the model using the initializer and create it using the save() method. The version is 2.

    user = User(name='name')
    user.save()

**Comment by saxix** [comment]
`IntegerVersionField` is not what you are looking for. Check [AutoIncVersionField](https://django-concurrency.readthedocs.io/en/latest/fields.html#autoincversionfield). Please consider that you should not add semantic to those values and ignore them, because concurrency is a low level system that should not be exposed to the business logic.

**Comment by shintophilip** [comment]
Yes. It was a mistake while copying the code. It is AutoIncVersionField.

**Comment by saxix** [comment]
yep, probably is true and I keep this issue opened as `enhancement`
